other/sr.js

```diff
@@ -0,0 +1,26 @@
+module.exports = () => ({ addUtilities }) => {
+  const sr = {
```

Comment:
i always add this one, curious why you classified as `other`?

Reply:
Yeah I always add it too, but it's one of those that falls under the 3rd bullet of "override a core plugin in Tailwind". On the other hand, maybe it's best to move it to **plugins**, and remove that 3rd bullet point stipulation. Just because there's a plugin in the plugins directory doesn't mean you have to `require` it in a project. 🤔

Reply:
also megan made some changes in FedPack that we should mirror here. once, this is finished, I can update fedpack to pull from here and eliminate the redundancy!

https://github.com/vigetlabs/FedPack/pull/118/files